# Created at 2022-11-20T10:46:43.616
Unexpected IOException with stream: 2022-11-20 10:46:43.615  INFO 8464 --- [stream-75814172] eConfig$$EnhancerBySpringCGLIB$$edba15a6 : STDERR: 2022-11-20T09:46:43.613926Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.


# Created at 2022-11-20T10:46:43.617
Unexpected IOException with stream: 2022-11-20 10:46:43.617  INFO 8464 --- [stream-75814172] eConfig$$EnhancerBySpringCGLIB$$edba15a6 : STDERR: 2022-11-20T09:46:43.615322Z 0 [System] [MY-013602] [Server] Channel mysql_main configured to support TLS. Encrypted connections are now supported for this channel.


# Created at 2022-11-20T10:46:43.631
Unexpected IOException with stream: 2022-11-20 10:46:43.631  INFO 8464 --- [stream-75814172] eConfig$$EnhancerBySpringCGLIB$$edba15a6 : STDERR: 2022-11-20T09:46:43.629954Z 0 [Warning] [MY-011810] [Server] Insecure configuration for --pid-file: Location '/var/run/mysqld' in the path is accessible to all OS users. Consider choosing a different directory.


# Created at 2022-11-20T10:46:44.016
Unexpected IOException with stream: 2022-11-20 10:46:44.016  INFO 8464 --- [stream-75814172] eConfig$$EnhancerBySpringCGLIB$$edba15a6 : STDERR: 2022-11-20T09:46:44.014381Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Bind-address: '::' port: 33060, socket: /var/run/mysqld/mysqlx.sock


# Created at 2022-11-20T10:46:44.021
Unexpected IOException with stream: 2022-11-20 10:46:44.019  INFO 8464 --- [stream-75814172] eConfig$$EnhancerBySpringCGLIB$$edba15a6 : STDERR: 2022-11-20T09:46:44.018760Z 0 [System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.28'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server - GPL.


# Created at 2022-11-20T10:46:44.151
Unexpected IOException with stream: 2022-11-20 10:46:44.151  INFO 8464 --- [           main] docker[mysql:8.0]                        : Container is started (JDBC URL: jdbc:mysql://localhost:55008/file_upload?createDatabaseIfNotExist=true&serverTimezone=UTC)


# Created at 2022-11-20T10:46:44.151
Unexpected IOException with stream: 2022-11-20 10:46:44.151  INFO 8464 --- [           main] docker[mysql:8.0]                        : Container mysql:8.0 started in PT48.013294S


# Created at 2022-11-20T10:46:44.164
Unexpected IOException with stream: 2022-11-20 10:46:44.164  INFO 8464 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Starting...


# Created at 2022-11-20T10:46:44.319
Unexpected IOException with stream: 2022-11-20 10:46:44.318  INFO 8464 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Start completed.


# Created at 2022-11-20T10:46:45.764
Unexpected IOException with stream: Running Changeset: db/db.file-upload-changelog.xml::1619276720000-1::dduru (generated)


# Created at 2022-11-20T10:46:45.919
Unexpected IOException with stream: Running Changeset: db/db.file-upload-changelog.xml::1634246089072-4::dduru (generated)


# Created at 2022-11-20T10:46:46.052
Unexpected IOException with stream: 2022-11-20 10:46:46.052  INFO 8464 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]


# Created at 2022-11-20T10:46:46.054
Unexpected IOException with stream: 2022-11-20 10:46:46.054  INFO 8464 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect


# Created at 2022-11-20T10:46:46.071
Unexpected IOException with stream: 2022-11-20 10:46:46.071  INFO 8464 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]


# Created at 2022-11-20T10:46:46.071
Unexpected IOException with stream: 2022-11-20 10:46:46.071  INFO 8464 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'


# Created at 2022-11-20T10:46:46.074
Unexpected IOException with stream: 2022-11-20 10:46:46.074  WARN 8464 --- [           main] o.s.w.c.s.GenericWebApplicationContext   : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'fileUploadProvider' defined in file [/Users/dduru/IdeaProjects/Private/libs/file-uploader-jpa/target/test-classes/com/julianduru/fileuploader/data/FileUploadProvider.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.julianduru.fileuploader.jpa.JpaFileUploadRepository' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}


# Created at 2022-11-20T10:46:46.074
Unexpected IOException with stream: 2022-11-20 10:46:46.074  INFO 8464 --- [           main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'


# Created at 2022-11-20T10:46:46.075
Unexpected IOException with stream: 2022-11-20 10:46:46.074  INFO 8464 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown initiated...


# Created at 2022-11-20T10:46:46.091
Unexpected IOException with stream: 2022-11-20 10:46:46.091  INFO 8464 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown completed.


